src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java

```diff
@@ -136,7 +128,7 @@
         "    \"AddressLine2\": \"Address Line 12\"\n" +
         "  },\n" +
         "  \"D8Document\":{" +
-        "    \"document_url\": \"http://localhost:%s/documents/05e7cd7e-7041-4d8a-826a-7bb49dfd83d0\"" +
+        "    \"document_url\": \"http://localhost:" + wireMockRule.port() + "/documents/05e7cd7e-7041-4d8a-826a-7bb49dfd83d0\"" +
```

Comment:
can we just have a method port()
which has the details on how to get the port? If that changes again in the future we would also have to change the implementation of that method